internals/generators/container/index.js

```diff
@@ -20,10 +20,10 @@ module.exports = {
     },
   }, {
     type: 'list',
-    name: 'component',
-    message: 'Select a base component:',
-    default: 'PureComponent',
-    choices: () => ['PureComponent', 'Component'],
+    name: 'type',
+    message: 'Select the base component type:',
+    default: 'Stateless Function',
```

Comment:
This will vary between project and developer (let's put it up for discussion) but I find that the 'Class Component' is by far the most popular way to define container components. Would love to hear what others do?